plugins/org.jkiss.dbeaver.net.ssh.ui/src/org/jkiss/dbeaver/ui/net/ssh/SSHTunnelConfiguratorUI.java

```diff
@@ -230,6 +232,25 @@ private static void setNumberEditStyles(Text text) {
         text.setLayoutData(gdt);
     }
 
+    @Nullable
+    private static DBPAuthInfo promptCredentialsDialog(@NotNull SSHConstants.AuthType type) {
+        final DBPAuthInfo credentialsToConnectTo;
+        try {
+            credentialsToConnectTo = DBWorkbench.getPlatformUI()
+                .promptUserCredentials("Please enter your SSH credentials",
+                    RegistryMessages.dialog_connection_auth_username,
```

Comment:
I think it is better to use SSHUIMessages here. And remove org.jkiss.dbeaver.registry.internal, from manifest after that.